go.mod

```diff
@@ -12,19 +12,18 @@ require (
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
 	github.com/openshift/custom-resource-status v0.0.0-20190822192428-e62f2f3b79f3
 	github.com/operator-backing-service-samples/postgresql-operator v0.0.0-20191023140509-5c3697ed3069
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20191115003340-16619cd27fa5
+	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200130164400-12c06cfc05c4
```

Comment:
Any specific reason `v0.14.1` can't be used as a version there?

Reply:
`go mod tidy` did this change.

Reply:
For more details read this FAQ: https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-package-that-has-not-opted-in-to-modules

Reply:
Raised an issue in OLM project: https://github.com/operator-framework/operator-lifecycle-manager/issues/1524